testing/assert.ts

```diff
@@ -0,0 +1,179 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import { assertEqual as prettyAssertEqual } from "./pretty.ts";
+
+interface Constructor {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  new (...args: any[]): any;
+}
+
+/** Make an assertion, if not `true`, then throw. */
+export function assert(expr: boolean, msg = ""): void {
+  if (!expr) {
+    throw new Error(msg);
+  }
+}
+
+/**
+ * Make an assertion that `actual` and `expected` are equal, deeply. If not
+ * deeply equal, then throw.
+ */
+export function equal(actual: unknown, expected: unknown, msg?: string): void {
```

Comment:
Rename to `assertEq`

Reply:
I only rename in this module not in the mod.ts `const assertions = { .... ` ? right?

Reply:
@zekth Sure keep the existing API in testing/mod.ts for now... but I'm going to remove that too and require people to import from here. But maybe that's better done in a separate commit.

Reply:
agreed.